### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by alrusdi**
Option to start server without 404 errors output to stdout. Start it like:
**npm run start -- hide-not-found-warnings**
Fix typo in russian localization
Show ruling party name on Turmoil board
Снимок экрана от 2020-06-29 20-45-35

### CODE REVIEW

**Comment by vincentneko** [comment]
Maybe it would be better not to hide some errors by default and therefore use a -- hide-not-found-warnings ?

**Comment by alrusdi** [comment]
> Maybe it would be better not to hide some errors by default and therefore use a -- hide-not-found-warnings ?

Done!

**Comment by vincentneko** [comment]
> > Maybe it would be better not to hide some errors by default and therefore use a -- hide-not-found-warnings ?
> 
> Done!

Thanks =)

File: server.ts [hunk 1]
```diff
@@ -805,7 +805,9 @@ function getGame(game: Game): string {
 }
 
 function notFound(req: http.IncomingMessage, res: http.ServerResponse): void {
-  console.warn('Not found', req.method, req.url);
+  if (process.argv.includes("hide-not-found-warnings")) {
+    console.warn('Not found', req.method, req.url);
```
Comment (vincentneko) [inline server.ts:809]: I think the if should be reversed

**Pull request merged**